sites/ledsmagazine.com/config/navigation.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
       { href: '/directory', label: 'Industry Guide' },
       { href: 'https://www.strategiesinlight.com/', label: 'Strategies in Light', target: '_blank' },
       { href: 'https://sapphireawards.secure-platform.com/a/', label: 'Awards', target: '_blank' },
-      { href: 'https://www.surveymonkey.com/r/leds40u40', label: '40 Under 40', target: '_blank' },
+      { href: '/leds-ssl-design/article/14179410/leds-magazine-40-under-40-class-features-leaders-across-led-and-ssl-sectors-magazine', label: '40 Under 40' },
```

Comment:
You may want to set the url to /14179410, because if the user changes the primary section or the name it would cause the alias to update and 404.